catt/http_server.py

```diff
@@ -5,18 +5,30 @@
 from pathlib import Path
 
 
-def serve_file(filename, address="", port=45114):
+def serve_file(filename, address="", port=45114, content_type=None):
     class FileHandler(BaseHTTPRequestHandler):
+
+        def format_size(self, size):
```

Comment:
What is this for?

Reply:
Very useful debugging information.
Now we get this kind of logs:
`10.1.1.3 - - [07/May/2018 11:53:56] "GET /?loaded_from_catt HTTP/1.1" 200 - video/mp4 - 39.80 MB`
`10.1.1.3 - - [07/May/2018 11:53:56] "GET //tmp/tmpmnfqm1im.vtt HTTP/1.1" 200 - text/vtt;charset=utf-8 - 125.97 KB`

I backported from my other PR.

I like knowing how big the file and the mime is useful for the other one anyway.